docker-compose.yml

```diff
@@ -1,5 +1,24 @@
 version: '3.8'
 
+x-app: &app
+  build:
+    context: .
+    target: hyku-base
+    args:
+      - EXTRA_APK_PACKAGES=less vim bash # TODO: this doesn't install vim, why?
```

Comment:
`hyku-base` doesn't seem to accept a `EXTRA_APK_PACKAGES` argument(?)

Reply:
it does not. I haven't looked in to it at all yet - do we have to do something to pass it back up a level?

Reply:
figured it out - they do not stack at all, so I added support